src/components/TooltipNew/style.scss

```diff
@@ -15,6 +15,20 @@
   text-align: left;
   transition-property: opacity, visibility;
 
+  &__close {
```

Comment:
you are repeating the same code two times. Can you store it in the mixin?
```suggestion
@mixin close-button {
  &__close {
    display: inline-block;
    position: absolute;
    right: 0;
    top: 0;
    padding: 8px;
    cursor: pointer;
    line-height: 0;
    
    svg {
      fill: currentColor;
    }
  }
}
```
or you can add the following line at the bottom of the file:
```
.popper-tooltip, css-tooltip {
  &__close {
    ...
  }
}
```

Reply:
done